.github/workflows/basic.yml

```diff
@@ -128,6 +62,16 @@ jobs:
     - name: Checkout
       uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
+    - name: Install oneAPI basekit
+      if: matrix.compiler.cxx == 'icpx'
+      run: |
+        sudo apt update
```

Comment:
nit: Use apt-get in scripts: https://github.com/oneapi-src/unified-memory-framework/actions/runs/9182058706/job/25250284899?pr=508#step:3:15